src/main/java/ch/yth2021/charjar/discord/module/friendly/FriendlyPeopleModule.java

```diff
@@ -0,0 +1,36 @@
+package ch.yth2021.charjar.discord.module.friendly;
+
+import ch.yth2021.charjar.API.User;
+import ch.yth2021.charjar.discord.module.BasicEventModule;
+import ch.yth2021.charjar.discord.module.FileReadHelper;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+
+public class FriendlyPeopleModule implements BasicEventModule {
+    private static final int POINTS_FOR_FRIENDLY_BEHAVIOUR = 10;
+    private static final String FILENAME = "friendlywords.txt";
+
+    @Override
+    public void onMessageReceived(MessageReceivedEvent event) {
+        String message = event.getMessage().getContentDisplay().toLowerCase();
+        if (messageContainsFriendlyWord(message)) {
+            User user = new User(event.getAuthor().getId());
+            try {
+                user.modPoints(POINTS_FOR_FRIENDLY_BEHAVIOUR);
```

Comment:
i feel like we should at least try to log errors
you can get the global logger with `LoggerFactory.getLogger(Application.class);`

Reply:
your comment does not cover the catch part but i guess one could improve that